sway-error/src/error.rs

```diff
@@ -2430,6 +2433,18 @@ impl ToDiagnostic for CompileError {
                         },
                 }
             },
+            ConfigurableMissingAbiDecodeInPlace { span } => Diagnostic {
+                reason: Some(Reason::new(code(1), "Configurables need a function named \"abi_decode_in_place\" to be in scope.".to_string())),
```

Comment:
Great to see expressive diagnostics being used :smile:
According to the guidelines, the "Reason" does not end with fullstop.
```suggestion
                reason: Some(Reason::new(code(1), "Configurables need a function named \"abi_decode_in_place\" to be in scope".to_string())),
```